src/docs/src/whatsnew/3.3.rst

```diff
@@ -0,0 +1,263 @@
+
+* :ghissue:`4106`: Set ``io_priority`` in all IO paths. Introduces ``system`` ``io_priority``.
+
+* :ghissue:`4144`, :ghissue:`4172`: Implement ``_bulk_get`` support for the replicator. Backward compatibility is ensured. This speeds up all replications. Add option to disable new behaviour for legacy setups.
+
+* :ghissue:`4163`: Statistically skip ``_revs_diff`` in the replicator. This improves performance for replications into empty targets.
+
+* :ghissue:`4177`: Remove the long deprecated ``bigcouch 0.4`` change sequence support.
+
+* :ghissue:`4238`: Optimize ``_bulk_get`` endpoint. This speeds up replication of 1M docs by ~2x. Individual ``_bulk_get`` requests are up to 8x faster.
+
+Bugfixes
+--------
+
+* :ghissue:`3817`: Fix undefined function call in ``weatherreport``.
+
+* :ghissue:`3819`: Return ``400`` instead of ``500`` response code for known invalid ``_bulk_docs`` with ``new_edits=false`` request.
+
+* :ghissue:`3861`: Add ``SameSite`` setting when clearing session cookies.
+
+* :ghissue:`3863`: Fix custom TLS distribution for Erlang 20.
+
+* :ghissue:`3870`: Always send all cookie attributes.
+
+* :ghissue:`3886`: Avoid changes feed rewind after shard move with no subsequent db updates.
+
+* :ghissue:`3888`: Make ``_stats`` endpoint resilient against nodes that go offline.
+
+* :ghissue:`3901`: Use db-creation time instead of ``0`` for ``instance_start_time`` to help replicator recognise whether a peer database was deleted and recreated.
+
+* :ghissue:`3909`: Fix ``new_edits:false`` and VDU ``function_clause``.
+
+* :ghissue:`3934`: Fix ``replicated_changes`` typo for purge doc updates.
+
+* :ghissue:`3940`: Ensure the multipart parser always monitors the worker and make sure to wait for attachment uploads before responding.
+
+* :ghissue:`3950`: Ignore responses from timed-out or retried ``ibrowse`` calls.
+
+* :ghissue:`3969`: Fix ``skip`` and ``limit`` for ``_all_dbs`` and ``_dbs_info``.
+
+* :ghissue:`3979`: Correctly respond with a ``500`` code when document updates time out under heavy load.
+
+* :ghissue:`3992`: Show that Search is available if it was available before. Avoid Search availability disappearing just because a Search node was temporarily not available.
+
+* :ghissue:`3993`: Return a ``400`` error when decoding a JWT token fails, rather htan crashing and not responding at all.
+
+* :ghissue:`3990`: Prevent creation of ddocs with no name through Mango index creation.
+
+* :ghissue:`4003`: Improve index building during shard splitting.
+
+* :ghissue:`4016`: Fix ``function_clause`` error for replicated changes with a target VDU.
+
+* :ghissue:`4020`: Fix ``maybe_handle_error`` clauses.
+
+* :ghissue:`4037`: Fix ES{256,384,512} support for JWTs.
+
+* :ghissue:`4040`: Handle ``exit(shutdown)`` error in ``chttpd``.
+
+* :ghissue:`4043`: Fix purge request timeouts (5s -> infinity).
+
+* :ghissue:`4046`: The ``devcontainer`` has been updated.
```

Comment:
#4046 is:  Add smoosh state diagram to directory and reference it.

#4146: Update devcontainer